js/src/components/contact-information/phone-number-card/phone-number-card.js

```diff
@@ -117,14 +117,27 @@ const PhoneNumberCard = ( {
 } ) => {
 	const { loaded, data } = phoneNumber;
 	const [ isEditing, setEditing ] = useState( initEditing );
+	const onPhoneNumberVerifiedRef = useRef();
+	onPhoneNumberVerifiedRef.current = onPhoneNumberVerified;
+
+	const { isVerified } = data;
 
 	// Handle the initial UI state of `initEditing = null`.
 	// The `isEditing` state is on hold. Determine it after the `phoneNumber` loaded.
 	useEffect( () => {
 		if ( loaded && isEditing === null ) {
-			setEditing( ! data.isValid );
+			setEditing( ! isVerified );
+		}
+	}, [ loaded, isVerified, isEditing ] );
+
+	// If `initEditing` is true, EditPhoneNumberCard takes care of the call of `onPhoneNumberVerified`
+	// after the phone number is verified. If `initEditing` is false or null, this useEffect handles
+	// the call of `onPhoneNumberVerified` when the loaded phone number has already been verified.
```

Comment:
❓  This was a bit confusing for me. 

Suggestion:

When `initEditing` prop is not true we call `onPhoneNumberVerified` when the loaded phone number has already been verified.

Reply:
It's intended to illustrate how `onPhoneNumberVerified` calls are handled under different conditions. I'd like to keep the explicit mention of "*If `initEditing` is false or null*" to clarify why the conditional below is a shorter one `initEditing ! == true` rather than a longer one `( initEditing === false || initEditing === null )`, and to avoid the appearance that it could be changed to `initEditing === false`.